playbooks/livy-spark3_kerberos_install.yml

```
@@ -0,0 +1,10 @@
+---
+- name: Kerberos Livy Server Install
+  hosts: livy_spark3_server
+  tasks:
+    - tosit.tdp.resolve:
+        node_name: livy_spark3_server
```

```suggestion
        node_name: livy-spark3_server
```
Service name does not respect the norm. The service is `livy-spark3` and component `server`.

With `livy_spark3_server`, I read service=Livy, component=spark3, action=server

---

Ok. Should I also change the name of the group? I think that `-` are now allowed in host group names